api/v1beta1/hostedcluster_types.go

```diff
@@ -158,6 +158,24 @@ const (
 	// allows the creation of guest clusters <= 4.13, which are before the rhcos kubevirt
 	// variant was released.
 	AllowUnsupportedKubeVirtRHCOSVariantsAnnotation = "hypershift.openshift.io/allow-unsupported-kubevirt-rhcos-variants"
+
+	// HostedClusterLabel is used as a label on nodes that are dedicated to a specific hosted cluster
+	HostedClusterLabel = "hypershift.openshift.io/cluster"
+
+	// RequestServingComponentLabel is used as a label on pods and nodes for dedicated serving components.
+	RequestServingComponentLabel = "hypershift.openshift.io/request-serving-component"
+
+	// IsolationModeAnnotation indicates the type of isolation should take effect for the
+	// hosted cluster's control plane workloads. Currently the only value supported is "dedicated-request-serving-components".
+	IsolationModeAnnotation = "hypershift.openshift.io/isolation-mode"
```

Comment:
just cosmetic but have curious if we have considered using the term topology here instead?
"hypershift.openshift.io/topology": shared / dedicated / dedicated-request-serving-components

Reply:
done